package.json

```diff
@@ -11,6 +11,9 @@
     "lint:fix": "eslint --fix . && prettier --write .",
     "test": "run-s build:* && run-s test:*",
     "test:e2e": "xvfb-maybe cross-env NODE_ENV=test playwright test 'test/e2e/.*.test.js'",
+    "vitestpreview": "vite preview",
+    "vitest": "vitest",
```

Comment:
this one needs to either be added to `npm test` or as a separate step to the actions file, so that it's integrated into CI. If you think it's right to run this on every test run, let's add it to `npm test`

Reply:
actually, it looks like `vitest` will stay open, waiting for changes. can we add a test run mode where it quits after it went through all the tests once?

Reply:
I think adding it to test script will work for now but it would be interesting to add it to a separated action step